javalib/src/main/scala/java/lang/process/UnixProcessGen2.scala

```diff
@@ -38,6 +38,27 @@ import scalanative.posix.time.timespec
 import scalanative.posix.timeOps.timespecOps
 import scalanative.posix.unistd
 
+/* A number of UnixProcessGen2 methods are synchronized where one
+ * would not expect such.  java.lang.Process was introduced in JDK 1.
+ * Even the most recent JDK 24 documentation of the class says nothing
+ * about the class being thread-safe. That means that one must treat
+ * it as concurrent access requiring external synchronization.
+ *
+ * So much for "de jure". "De facto" applications such as Li Hayoi's os-lib
```

Comment:
```suggestion
 * So much for "de jure". "De facto" applications such as Li Haoyi's os-lib 
```

Reply:
Eric, thank you for catching that.

Is the quote by W.C. Fields, of Philadelphia fame, to newspaper reporters: "Call me anything you want
but spell my name right."

I'll correct this later tonight when I have access to my primary machine (even a typo needs re-building).